src/components/MainLayout/styles.module.css

```diff
@@ -2,3 +2,11 @@
   font-weight: 400 !important;
   background-color: #fff;
 }
+
+.skipLink {
+  &:focus {
+    z-index: 999;
+    font-size: 1.125rem;
```

Comment:
Should we improve the styling a bit? Right now, it's just a white rectangle holding a non decorated link

<img width="213" alt="image" src="https://user-images.githubusercontent.com/43496356/160145604-e37aa7fb-97c2-4c94-9e2a-7f1bedeea0a9.png">

Maybe add some border-radius, box-shadow, and link styling?